core/src/main/kotlin/com/gitlab/kordlib/core/entity/Invite.kt

```diff
@@ -146,4 +144,20 @@ data class Invite(
     override fun withStrategy(strategy: EntitySupplyStrategy<*>): Invite =
             Invite(data, kord, strategy.supply(kord))
 
+    companion object {
+        fun from(invite: InviteResponse) = with(invite) {
```

Comment:
Pretty sure this doesn't belong here

Reply:
Yeah. I'm caching from rest. :sweat_smile: